backup-commons/src/main/java/com/opsgenie/tools/backup/util/BackupUtils.java

```diff
@@ -46,6 +47,20 @@ public static String readFile(String fileName) throws IOException {
 
         return sb.toString();
     }
+    public static String getTeamNameFromId(File teamsDirectory, String teamId) {
+        String[] files = getFileListOf(teamsDirectory);
```

Comment:
can you check fix `getFileListOf` function to return empty list instead of null.